core/src/com/biglybt/core/networkmanager/VirtualChannelSelector.java

```diff
@@ -41,7 +41,7 @@
   public static final int OP_READ   = SelectionKey.OP_READ;
   public static final int OP_WRITE  = SelectionKey.OP_WRITE;
 
-  private boolean SAFE_SELECTOR_MODE_ENABLED = TEST_SAFE_MODE || COConfigurationManager.getBooleanParameter( "network.tcp.enable_safe_selector_mode" );
+  private boolean SAFE_SELECTOR_MODE_ENABLED = TEST_SAFE_MODE;
```

Comment:
There isn't any way to enable "safe selector mode" anymore. 
I don't believe there's any valid reason to keep the option to test it.. is there?